Readme.md

```diff
@@ -14,15 +14,15 @@ This is a fork of the [bytes][bytes] module, except:
 
  * It uses IEC units by default
  * Supports a wider range of units
- * Supports changing to compatability ([JEDEC][jedec]) mode, and formatting in whichever prefix type you prefix (binary, metric, jedec)
+ * Supports changing to compatibility ([JEDEC][jedec]) mode, and formatting in whichever unit type you prefer (binary, metric, jedec)
```

Comment:
prefix here actually refers to the "mega" or "mebi" which are prefixes… but yours is more clear in the end

Reply:
Ah yeah, that makes sense, sorry I didn't catch it